src/core/connection.c

```diff
@@ -7277,6 +7302,28 @@ QuicConnParamGet(
         Status = QUIC_STATUS_SUCCESS;
         break;
 
+    case QUIC_PARAM_CONN_QTIP:
+        if (QuicConnIsServer(Connection)) {
+            Status = QUIC_STATUS_INVALID_STATE;
+            break;
+        }
+        if (*BufferLength < sizeof(BOOLEAN)) {
+            *BufferLength = sizeof(BOOLEAN);
+            Status = QUIC_STATUS_BUFFER_TOO_SMALL;
+            break;
+        }
+
+        if (Buffer == NULL) {
+            Status = QUIC_STATUS_INVALID_PARAMETER;
+            break;
+        }
+
+        *BufferLength = sizeof(BOOLEAN);
```

Comment:
nit: Consider sizeof(Connection->State.UseQTIP)

Reply:
ACK

Reply:
Actually, updating it to sizeof(Connection->State.UseQTIP) leads to a compilation error (I think it has to do with the way the structure is defined).

Gonna close this thread for now. Feel free to re-open if needed.